src/main/java/org/sonar/plugins/stash/CoverageCompat.java

```diff
@@ -0,0 +1,21 @@
+package org.sonar.plugins.stash;
+
+import org.sonar.api.issue.Issue;
+
+public final class CoverageCompat {
+  private CoverageCompat() {}
+
+  private final static String repo = "coverageEvolution";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)